.github/workflows/build-tool-windows.yml

```diff
@@ -1,6 +1,13 @@
-name: Build-Tools
+name: Build-Tools-Windows
 
-on: [push, pull_request]
+on:
+  # It is designed to only fire if the tools change.
+  push:
+    paths:
+      - 'tools/**'
+  pull_request:
+    paths:
+      - 'tools/**'
```

Comment:
I’d rather not merge this. It makes an assumption about the absence of dependencies which is wrong in the unstable branch (the new mksprite depends on sprite.h, and rdpvalidate has dependencies too), and I don’t want to remember to update here any time we add one more. The workflow is quite fast anyway

Reply:
Ok, I will revert that bit for the moment.

Reply:
Fixed.